app/code/Magento/Catalog/Test/Mftf/Test/AdminSortingByWebsitesTest.xml

```diff
@@ -0,0 +1,85 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+ /**
+  * Copyright © Magento, Inc. All rights reserved.
+  * See COPYING.txt for license details.
+  */
+-->
+
+<tests xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+       xsi:noNamespaceSchemaLocation="urn:magento:mftf:Test/etc/testSchema.xsd">
+    <test name="AdminSortingByWebsitesTest">
+        <annotations>
+            <stories value="View sorting by websites"/>
+            <title value="Sorting by websites in Admin"/>
+            <description value="Sorting products by websites in Admin"/>
+        </annotations>
+        <before>
+            <actionGroup ref="LoginAsAdmin" stepKey="loginAsAdmin"/>
+            <!--Create new website -->
+            <actionGroup ref="AdminCreateWebsiteActionGroup" stepKey="createAdditionalWebsite">
+                <argument name="newWebsiteName" value="Second Website"/>
+                <argument name="websiteCode" value="second_website"/>
```

Comment:
I would recommend to use data entity `customWebsite`.
It will create unique WebSite Name and WebSite code.
Arguments will look like this
```
<argument name="newWebsiteName" value="{{customWebsite.name}}"/>
<argument name="websiteCode" value="{{customWebsite.code}}"/>
```

Reply:
done